NanoServerPackage/Test/NanoServerPackage.Save.Tests.ps1

```diff
@@ -59,6 +59,31 @@ Describe "Save-WindowsPackage Stand-Alone" {
         Remove-Item $pathToSave\*.cab
     }
 
+    It "Save-NanoServerPackage Name, Path, Force" {
+        $name = $names | Get-Random
+        $version = $requiredVersion
+
+        $SavePath = "$TestDrive\SaveFolder"
```

Comment:
Pester ensures that $TestDrive is clean

---
In reply to: [82315387](https://github.com/OneGet/NanoServerPackage/pull/6#discussion_r82315387) [](ancestors = 82315387)